### DISCUSSION TITLE
Update sponsors on the website

### DISCUSSION THREAD

**Discussion by AB-xdev** [testcontainers/testcontainers-java#8762] [General]
The sponsors on the website seem to be a bit outdated...
[We](https://github.com/xdev-software) are e.g. sponsoring since 2022 and there is no mention of us despite the sponsorship having the following statement:
> As a backer, your name will be listed on the front page of testcontainers.org as a small token of our gratitude.

PS: Some kind of email or something similar for direct contact on the website would also be appreciated so I don't have to open issues here...

**Comment by kiview** [comment]
Hey @AB-xdev, sorry, I somehow forgot to send my answer to the discussion. We've updated https://testcontainers.com/, XDEV Software is now listed under Backers. Thank your for the ping and sorry for missing this.

Best way to contact us, is our Testcontainers Slack.
